packages/protocol/scripts/truffle/releaseGoldContracts.json

```diff
@@ -0,0 +1,20 @@
+{
+  "entity": "0x000000000000000000000000000000000000ce12",
+  "grants": [
+    {
+    "releaseStartTime": "02-24-2020",
+    "releaseCliffTime": 1,
+    "numReleasePeriods": 1,
+    "releasePeriod": 100000000,
+    "amountReleasedPerPeriod": 1,
+    "revocable": true,
+    "beneficiary": "0x000000000000000000000000000000000000ce11",
+    "releaseOwner": "0x000000000000000000000000000000000000ce11",
+    "refundAddress": "0x000000000000000000000000000000000000ce11",
+    "subjectToLiquidityProvision": true,
+    "initialDistributionPercentage": 1000,
```

Comment:
nit: "Percentage" is confusing here since we add an extra decimal of precision

Reply:
Ratio? factor? Can't think of a good word for this